<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Node</string>
  </property>
  <widget class="QLabel" name="labelMethod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Method:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoxType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Box type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropMethod">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropBoxType">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
